packages/components/pill/src/Pill.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { cx } from 'emotion';
+import { CommonProps, Box } from '@contentful/f36-core';
+import { TabFocusTrap } from '@contentful/f36-utils';
+import { Drag, Close } from '@contentful/f36-icons';
+import { PillVariants } from './types';
+import getPillStyles from './styles';
+
+export interface PillProps
+  extends React.HTMLAttributes<HTMLElement>,
+    CommonProps {
+  /**
+   * Text that will be shown on the pill
+   */
+  label: string;
+  /**
+   * Function that handles when the close icon is clicked. Close icon visibility also depends on if this is set
```

Comment:
Minor thing, I would rephrase here a bit: 
(... ) Close icon visibility depends on if this property is set.
Now when I think about it, it would be good to maybe extend to code styleguide with guidelines around comments. Or it's overkill?

Reply:
yeah, that phrasing looks better, will update it for now, and we can talk about the guidelines for comments later, to have it consistent across all components.